sdks/python/apache_beam/runners/dataflow/internal/apiclient.py

```diff
@@ -923,8 +923,9 @@ def job_id_for_name(self, job_name):
           pageToken=token)
       response = self._client.projects_locations_jobs.List(request)
       for job in response.jobs:
-        if (job.name == job_name and job.currentState ==
-            dataflow.Job.CurrentStateValueValuesEnum.JOB_STATE_RUNNING):
+        if (job.name == job_name and job.currentState in
```

Comment:
Could you add a pydoc comment to this function explaining that this is used for updates and it will return a job id only for running/draining pipelines?

Should we change the function name, this seems to be a internal only function?

Reply:
Since this is only used by update, I'd support renaming the method to include 'running'.

Reply:
On a second thought, I do not know if we will break some users if we rename this. I do not think so but it will still a breaking change.